src/engine/target.js

```diff
@@ -322,6 +322,12 @@ class Target extends EventEmitter {
                         this.runtime.ioDevices.cloud.requestRenameVariable(oldName, newName);
                     }
 
+                    if (variable.type === '') {
```

Comment:
```suggestion
                    if (variable.type === Variable.SCALAR_TYPE) {
```

Reply:
Also add a note that we're only doing this for variables because lists don't exist in the "sensing_of" dropdown.

Although `updateVariableName` could *theoretically* be expanded in the future to blocks which could reference lists... Currently, there's nothing in `updateVariableName`'s documentation that indicates that it only does things to `sensing_of`blocks.